File: VB/CustomDrawingSample/Model/Order.vb
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema

Namespace CustomDrawingSample.Model

    Public Partial Class Order

        <Key>
        Public Property OrderID As Integer

        <Column("EmployeeID")>
        <ForeignKey("Employee")>
        Public Property EmployeeId As Integer?

        <Column("Freight", TypeName:="smallmoney")>
        Public Property Freight As Decimal

        <Column("OrderDate", TypeName:="datetime")>
        Public Property OrderDate As Date

        Public Overridable Property Employee As Employee
    End Class
End Namespace

File: CS/CustomDrawingSample/Model/Order.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace CustomDrawingSample.Model {
    public partial class Order {
        [Key]
        public int OrderID { get; set; }

        [Column("EmployeeID")]
        [ForeignKey("Employee")]
        public int? EmployeeId { get; set; }

        [Column("Freight", TypeName = "smallmoney")]
        public decimal Freight { get; set; }

        [Column("OrderDate", TypeName = "datetime")]
        public DateTime OrderDate { get; set; }

        public virtual Employee Employee { get; set; }
    }
}
